test/container/test_helper_scripts.py

```diff
@@ -19,6 +19,14 @@ def test_php_images_contain_helper_scripts(host):
         assert host.file(file).is_file is True
         assert host.file(file).mode == 0o775
 
+@pytest.mark.php_dev
+def test_php_images_contain_dev_helper_scripts(host):
+    helper_script = '/usr/local/bin/docker-php-dev-mode'
```

Comment:
you can rewrite this to:
```python
    file = host.file('/usr/local/bin/docker-php-dev-mode')

    assert file.is_file is True
    assert file.mode == 0o775
```

Since you're using a variable anyway, also exists and is_file might be redundant (maybe you copied from my tests, but still..)